---
/mission_tunnel:
  ros__parameters:
    init_pose:
      position:
        x: 0.0
        y: 0.0
        z: 0.0
      orientation:
        x: 0.0
        y: 0.0
        yaw: 0
    goal_pose:
      position:
        x: 1.4
        y: -1.4
        z: 0.0
      orientation:
        x: 0.0
        y: 0.0
        yaw: 0
